### PR TITLE
Fixes #33336 - New CV UI - Affected repositories tab

### PR DESCRIPTION
**Pull request by sjha4**

### CODE REVIEW

File: webpack/scenes/ContentViews/Details/ContentViewDetails.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ const ContentViewDetails = ({ match }) => {
               </FlexItem>
             </Flex>
           </GridItem>
-          <GridItem span={4}>
+          <GridItem span={3}>
             <Flex>
```
Comment (Andrewgdewar) [inline webpack/scenes/ContentViews/Details/ContentViewDetails.js:90]:
```suggestion
          <GridItem xl={4} lg={5} sm={12} >
            <Flex justifyContent={{ lg: 'justifyContentFlexEnd', sm: 'justifyContentFlexStart' }}>
```

File: webpack/scenes/ContentViews/Details/ContentViewDetails.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ const ContentViewDetails = ({ match }) => {
               {name || <Skeleton />}
             </BreadcrumbItem>
           </Breadcrumb>
-          <GridItem span={8}>
+          <GridItem span={9}>
```
Comment (Andrewgdewar) [inline webpack/scenes/ContentViews/Details/ContentViewDetails.js:77]:
```suggestion
           <GridItem xl={8} lg={7} sm={12} >
```

**Review by Andrewgdewar — APPROVED** [review]
Ack.

**Review by chris1984 — APPROVED** [review]
ACK, tested and works great. @Andrewgdewar acked it from a code perspectivce

**Pull request merged**